packages/chakra-ui/src/FieldTemplate/FieldTemplate.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { FormHelperText, FormControl, FormErrorMessage } from '@chakra-ui/react'
+import { FormLabel as WrappedFormLabel } from '@chakra-ui/react'
+import { FieldTemplateProps } from '@rjsf/core'
+
+const FieldTemplate = ({ id, children, label, displayLabel, disabled, readonly, required, rawErrors = [], rawHelp }: FieldTemplateProps) => (
+  <FormControl isDisabled={disabled} mt={4} isReadOnly={readonly} isRequired={required} isInvalid={!!rawErrors.length}>
```

Comment:
Can we add a bit less margin-top here, maybe try `2`? The form seems too spaced out:

![image](https://user-images.githubusercontent.com/1689183/111926953-49e59c00-8a85-11eb-8a46-8b3be37a7c78.png)


Compare this to the default theme:

![image](https://user-images.githubusercontent.com/1689183/111926960-4fdb7d00-8a85-11eb-806f-0f7194e734bf.png)